extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -64,8 +64,8 @@ private static function renderImage( $media, $id ) {
 					'url' => $media['url'],
 					'title' => $media['title'],
 					'fileUrl' => $media['fileUrl'],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "fileUrl" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)